text/src/autogluon/text/automm/utils.py

```diff
@@ -23,6 +23,9 @@
     CategoricalMLP,
     NumericalMLP,
     MultimodalFusionMLP,
+    NumericalTransformer,
+    CategoricalTransformer,
```

Comment:
What is the reason to split numeric and categorical transformers into two separate models vs doing all features in one model?

Reply:
What is implemented here will be a superset of the original FTTransformer. If we set depth=0 for NumericalTransformer and CategoricalTransformer and set depth=3 for FusionTransformer, it will be the same as the architecture proposed by the paper. However, we can also choose to set `depth=1` for categorical / numerical layers to enhance the fusion of features from certain modalities..